.travis.yml

```diff
@@ -3,14 +3,20 @@ python:
 - '3.5'
 - '3.6'
 env:
-- DJANGO=1.11
-- DJANGO=2.0
-- DJANGO=2.1
+- SOLR_VERSION=6.6.5
+- DJANGO=1.11 SOLR_VERSION=6.6.5
```

Comment:
The repeated Solr version env seems redundant, is this for future versions when we test against multiple versions of Solr?

Reply:
Exactly for that. I wanted to make sure we had that built in early on, and that the script that loads Solr was looking in exactly one canonical place for the Solr version to install.

Reply:
I checked the travis docs and saw that there doesn't seem to be an easy way to set up a test matrix based on environment variables.

I figured out after I looked at this (but never came back to comment) that the first line is testing without django. Maybe worth a comment to make it explicit we're testing without django there, Just so people scanning quickly catch it.